<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeEventControlerWidget</class>
 <widget class="QWidget" name="TimeEventControlerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimeEventControlerWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#timeEventControler, QFrame#timeEventBar {
  background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.05 rgba(255, 255, 255, 255));
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,10">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="timeEventControler">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>110</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="prevEventButton">
            <property name="toolTip">
             <string>Previous event</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:flat {
     border: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playButton">
            <property name="toolTip">
             <string>Play / pause</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:flat {
	border: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextEventButton">
            <property name="toolTip">
             <string>Next event</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:flat {
     border: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>14</width>
              <height>14</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="numDigits">
           <number>7</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="acquisitionOptionsButtonMenu">
            <property name="toolTip">
             <string>Acquisition options</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:flat {
     border: none;
}

QPushButton::menu-indicator {
     image: none;
}</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/Resources/Images/acquisition_options.png</normaloff>:/Resources/Images/acquisition_options.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="eventOptionsButtonMenu">
            <property name="toolTip">
             <string>Event options</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton:flat {
     border: none;
}

QPushButton::menu-indicator {
     image: none;
}</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/Resources/Images/event_options.png</normaloff>:/Resources/Images/event_options.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="eventRightOptionsButtonMenu">
          <property name="toolTip">
           <string>Right event options</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:flat {
     border: none;
}

QPushButton::menu-indicator {
     image: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/Resources/Images/event_right.png</normaloff>:/Resources/Images/event_right.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>16</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eventLeftOptionsButtonMenu">
          <property name="toolTip">
           <string>Left event options</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:flat {
     border: none;
}

QPushButton::menu-indicator {
     image: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/Resources/Images/event_left.png</normaloff>:/Resources/Images/event_left.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>16</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eventGeneralOptionsButtonMenu">
          <property name="toolTip">
           <string>General event options</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:flat {
     border: none;
}

QPushButton::menu-indicator {
     image: none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/Resources/Images/event_general.png</normaloff>:/Resources/Images/event_general.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>16</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>15</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="TimeEventBarWidget" name="timeEventBar">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionPlaybackSpeedRealtime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Realtime</string>
   </property>
  </action>
  <action name="actionPlaybackSpeed1_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1/2</string>
   </property>
  </action>
  <action name="actionPlaybackSpeed1_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1/4</string>
   </property>
  </action>
  <action name="actionPlaybackSpeed1_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1/5</string>
   </property>
  </action>
  <action name="actionPlaybackSpeed1_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1/10</string>
   </property>
  </action>
  <action name="actionRemoveSelectedEvents">
   <property name="text">
    <string>Remove selected events</string>
   </property>
  </action>
  <action name="actionCropRegionOfInterest">
   <property name="text">
    <string>Crop Frames of Interest</string>
   </property>
  </action>
  <action name="actionClearEvents">
   <property name="text">
    <string>Clear events</string>
   </property>
  </action>
  <action name="actionInsertRightFootStrike">
   <property name="text">
    <string>Insert Right Foot Strike</string>
   </property>
  </action>
  <action name="actionInsertRightFootOff">
   <property name="text">
    <string>Insert Right Foot Off</string>
   </property>
  </action>
  <action name="actionInsertRightOther">
   <property name="text">
    <string>Insert Right ...</string>
   </property>
  </action>
  <action name="actionRemoveAllRightFootStrike">
   <property name="text">
    <string>Remove all Right Foot Strike</string>
   </property>
  </action>
  <action name="actionRemoveAllRightFootOff">
   <property name="text">
    <string>Remove all Right Foot Off</string>
   </property>
  </action>
  <action name="actionInsertLeftFootStrike">
   <property name="text">
    <string>Insert Left Foot Strike</string>
   </property>
  </action>
  <action name="actionInsertLeftFootOff">
   <property name="text">
    <string>Insert Left Foot Off</string>
   </property>
  </action>
  <action name="actionInsertLeftOther">
   <property name="text">
    <string>Insert Left ...</string>
   </property>
  </action>
  <action name="actionRemoveAllLeftFootStrike">
   <property name="text">
    <string>Remove all Left Foot Strike</string>
   </property>
  </action>
  <action name="actionRemoveAllLeftFootOff">
   <property name="text">
    <string>Remove all Left Foot Off</string>
   </property>
  </action>
  <action name="actionInsertGeneralFootStrike">
   <property name="text">
    <string>Insert Foot Strike</string>
   </property>
  </action>
  <action name="actionInsertGeneralFootOff">
   <property name="text">
    <string>Insert Foot Off</string>
   </property>
  </action>
  <action name="actionInsertGeneralOther">
   <property name="text">
    <string>Insert ...</string>
   </property>
  </action>
  <action name="actionRemoveAllGeneralFootStrike">
   <property name="text">
    <string>Remove all General Foot Strike</string>
   </property>
  </action>
  <action name="actionRemoveAllGeneralFootOff">
   <property name="text">
    <string>Remove all General Foot Off</string>
   </property>
  </action>
  <action name="actionEditSelectedEvents">
   <property name="text">
    <string>Edit selected events</string>
   </property>
  </action>
  <action name="actionZoomUnzoomRegionOfInterest">
   <property name="text">
    <string>Zoom Frames of Interest</string>
   </property>
  </action>
  <action name="actionNextEvent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Next Event</string>
   </property>
  </action>
  <action name="actionPreviousEvent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Previous Event</string>
   </property>
  </action>
  <action name="actionReframeFromOne">
   <property name="text">
    <string>Reframe From One</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeEventBarWidget</class>
   <extends>QFrame</extends>
   <header>TimeEventBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
